sdk/python/feast/on_demand_feature_view.py

```diff
@@ -51,8 +48,7 @@ class OnDemandFeatureView(BaseFeatureView):
             sources with type FeatureViewProjection.
         source_request_sources: A map from input source names to the actual input
             sources with type RequestSource.
-        udf: The user defined transformation function, which must take pandas dataframes
```

Comment:
can we still keep the `udf` and add `deprecated` to make it still backward compatible? I like the new field `transformation` but just in case some teams still use `udf` in the code base

Reply:
Sure, that's a good idea

Reply:
Done

Reply:
Cool. nit, maybe keep the `udf` in the comment as well.

Reply:
It's in the comment for the init method. Do you mean the comment that lists the attributes? it's no longer an attribute, so probably shouldn't be there. It's being converted to transformation in init method and discarded.

Reply:
ah that make sense